content/geometry/HullDiameter.h

```diff
@@ -3,15 +3,17 @@
  * Date: 2019-05-05
  * License: CC0
  * Source: https://codeforces.com/blog/entry/48868
- * Description: Returns the two points with max distance on a convex hull (no duplicate/collinear points).
- * If input isn't a convex hull, call "convexHull" first.
+ * Description: Returns the two points with max distance on a convex hull (CCW,
+ * no duplicate/collinear points). If input isn't a convex hull, call
+ * "convexHull" first.
  * Status: Fuzz-tested, submitted on kattis roberthood
  */
 #pragma once
 
 typedef Point<ll> P;
 array<P, 2> hullDiameter(vector<P> S) {
 	int n = sz(S), j = n < 2 ? 0 : 1;
+	assert(n>=2);
```

Comment:
space around `>=`, and you can set `j = 1` unconditionally given this

Reply:
Actually, I'm back to thinking that it makes marginally more sense if `{S[0], S{0]}` is returned.